recipes/arc3o/LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
You can remove this license file since one is packaged in the source distribution.

Reply:
thanks! done!